pkg/cli/main.go

```diff
@@ -47,6 +48,10 @@ func Start(rawArgs []string, streams *commands.Streams) (int, error) {
 	}
 	flag.CommandLine.Parse(rawArgs)
 
+	if *flConfig != "" {
+		os.Setenv("SCW_CONFIG_PATH", *flConfig)
```

Comment:
I think you should not set the environment variable here, but instead return a default if `SCW_CONFIG_PATH` is not set below in `pkg/config/config.go`.